opensuse_buildservice/notqmail.spec

```diff
@@ -0,0 +1,423 @@
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm2+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm3
+%attr(0755,root,qmail)            %{qmaildir}/boot/home+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm1
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm2
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm1+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm3+df
+%if "%{?_unitdir}" != ""
+                                  %{_unitdir}/%{name}.service
+%endif
+
+%if %noperms == 0
+%if 0%{?suse_version} >= 1120
+%attr(644,root,root)                               %{_sysconfdir}/permissions.d/%{name}-permissions
+%attr(644,root,root)                               %{_sysconfdir}/permissions.d/%{name}-permissions.secure
+%endif
+%endif
+%if %noperms == 0
+%if 0%{?suse_version} >= 1120
+%verify (not user group mode) %attr(6551, qmailq, qmail)   %{qmaildir}/bin/qmail-queue
+%verify (not user group mode) %attr(2555, alias, qmail)    %{qmaildir}/alias
+%endif
+%endif
+
+%files doc
+%{_mandir}/man?/*%{?ext_man}
+%{nq_docdir}
+
+### SCRIPTLETS ###############################################################################
+%verifyscript
+ID=$(id -u)
+if [ $ID -ne 0 ] ; then
+  echo "You are not root" 1>&2
+  exit 1
+fi
+
+%if %noperms == 0
+%if 0%{?suse_version} >= 1120
+%verify_permissions -e %{qmaildir}/sbin/qmail-queue
+%verify_permissions -e %{qmaildir}/alias
+%endif
+%endif
+
+%pretrans
+argv1=$1
+ID=$(id -u)
+if [ $ID -ne 0 ] ; then
+  echo "You are not root" 1>&2
+  exit 1
+fi
+if test -f %{_sysconfdir}/systemd/system/multi-user.target.wants/notqmail.service
+then
+  echo "Giving %{name} exactly 5 seconds to exit nicely"
+  /bin/systemctl stop notqmail > /dev/null 2>&1
+fi
+sleep 5
+
+%if 0%{?fedora_version} || 0%{?centos_version} || 0%{?rhel_version} || 0%{?suse_version} < 1500 || 0%{?sles_version} < 15
+%pre
+/usr/bin/getent group nofiles   > /dev/null || /usr/sbin/groupadd nofiles
+/usr/bin/getent group qmail     > /dev/null || /usr/sbin/groupadd qmail
+/usr/bin/getent passwd alias    > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}/alias  -s /sbin/nologin alias
+/usr/bin/getent passwd qmaild   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}        -s /sbin/nologin qmaild
+/usr/bin/getent passwd qmaill   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{logdir}          -s /sbin/nologin qmaill
+/usr/bin/getent passwd qmailp   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}        -s /sbin/nologin qmailp
+/usr/bin/getent passwd qmailq   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmailq
+/usr/bin/getent passwd qmailr   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmailr
+/usr/bin/getent passwd qmails   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmails
+for i in alias qmaild qmaill qmailp qmailq qmailr qmails qscand
```

Comment:
qscand?

Reply:
Oops. Result of copy paste. This has to go away